benchmark/solkz/compositional_field/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ ENDIF ()
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 SET(TARGET "solkz_compositional_field")
+SET(TARGET_DEPENDENCY "solkz")
+
 PROJECT(${TARGET})
 
+INCLUDE_DIRECTORIES("..")
+
 ADD_LIBRARY(${TARGET} SHARED solkz_compositional_field.cc)
-ASPECT_SETUP_PLUGIN(${TARGET})	
+ADD_LIBRARY(${TARGET_DEPENDENCY} SHARED ../solkz.cc)
```

Comment:
I have never seen this syntax. Generally, at compile time one includes the flag 
`-I$ASPECT_SOURCE_DIR/benchmarks/solkz` and then possibly `#include "solkz.h` but then again, my experience is limited. Thanks for the pointer, I'll see what I can cook up, tomorrow that is. :)